modules/olmv1-updating-an-operator.adoc

```diff
@@ -159,6 +161,53 @@ spec:
 ----
 <1> Update the version from `1.11.1` to `1.12.1`
 
+** If you want to define a range of acceptable update versions, edit your CR similar to the following example:
++
+.Example CR with a version range specified
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  version: ">1.11.1, <1.13" <1>
+----
+<1> Specifies that the desired version range is greater than version `1.11.1` and less than `1.13`. For more information, see "Support for version ranges" and "Version comparison strings".
+
+** If you want to update to the latest version that can be resolved from a channel, edit your CR similar to the following example:
++
+.Example CR with a specified channel
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  channel: pipelines-1.13 <1>
+----
+<1> Installs the latest release that can be resolved from the specified channel. Updates to the channel are automatically installed.
+
+** If you want to specify a channel and version or version range, edit your CR similar to the following example:
++
+.Example CR with a specified channel
+[source,yaml]
+----
+apiVersion: operators.operatorframework.io/v1alpha1
+kind: Operator
+metadata:
+  name: pipelines-operator
+spec:
+  packageName: openshift-pipelines-operator-rh
+  channel: latest
```

Comment:
IMO this example might need a placeholder and corresponding callout to highlight which field specifies the channel?

Reply:
Heard. I did a draft of the doc where I took that approach. But the goal here is to give an actual example from the RH Operators catalog. So, using a placeholder goes against the goal of showing a customer a working CR.